src/machine/board_lgt92.go

```diff
@@ -0,0 +1,107 @@
+// +build lgt92
+
+package machine
+
+import (
+	"device/stm32"
+	"runtime/interrupt"
+)
+
+const (
+	LED1 = PA12
+	LED2 = PA8
+	LED3 = PA11
+
+	LED_RED   = LED1
+	LED_BLUE  = LED2
+	LED_GREEN = LED3
+
+	BUTTON = PB14
+
+	// LG GPS module
+	GPS_STANDBY_PIN = PB3
+	GPS_RESET_PIN   = PB4
+	GPS_POWER_PIN   = PB5
+
+	// LORA
+	RFM95_DIO0_PIN = PC13
+
+	//TinyGo UART is MCU LPUSART1
+	UART_RX_PIN = PA13
+	UART_TX_PIN = PA14
+
+	//TinyGo UART1 is MCU USART1
+	UART1_RX_PIN = PB6
+	UART1_TX_PIN = PB7
+)
+
+var (
+
+	// Console UART (LPUSART1)
+	UART0 = UART{
+		Buffer:          NewRingBuffer(),
+		Bus:             stm32.LPUSART1,
+		AltFuncSelector: 6,
+	}
+
+	// Gps UART
+	UART1 = UART{
+		Buffer:          NewRingBuffer(),
+		Bus:             stm32.USART1,
+		AltFuncSelector: 0,
+	}
+)
+
+// set up RX IRQ handler. Follow similar pattern for other UARTx instances
+func init() {
+
+	// Enable GPIOs
+	stm32.RCC.IOPENR.SetBits(stm32.RCC_IOPENR_IOPAEN)
+	stm32.RCC.IOPENR.SetBits(stm32.RCC_IOPENR_IOPBEN)
+	stm32.RCC.IOPENR.SetBits(stm32.RCC_IOPENR_IOPCEN)
+
+	// Enable NVM (eeprom)
+	stm32.RCC.AHBENR.SetBits(stm32.RCC_AHBENR_MIFEN)
+
+	// Enable Leds GPIO output
+	LED_RED.Configure(PinConfig{Mode: PinOutput})
+	LED_GREEN.Configure(PinConfig{Mode: PinOutput})
+	LED_BLUE.Configure(PinConfig{Mode: PinOutput})
+
+	LED_RED.Low()
+	LED_GREEN.Low()
+	LED_BLUE.Low()
```

Comment:
Why are you doing this in the board file?

- Configuring outputs needs to be done by the program, unless it's really necessary they should be configured by a program that wants to use these LEDs.
- Is there any support for EEPROM? If not, I would suggest leaving it disabled (enabling unused peripherals may cost power).
- If you need to enable the GPIO peripheral (`stm32.RCC.IOPENR.SetBits(stm32.RCC_IOPENR_IOPAEN)`), this should be done in `Pin.Configure`. It looks like this is already done, so these lines are unnecessary.